cloud-formation/scripts/post-dev-stack-creation.sh

```diff
@@ -3,6 +3,8 @@
 source ./utils.sh
 source ./stack-name.sh
 
+export AWS_PROFILE=media-service
```

Comment:
Cool.

This has just made me realise `AWS_DEFAULT_PROFILE` is [legacy](https://github.com/aws/aws-cli/issues/1281#issuecomment-92509459) too! (We mention setting `AWS_DEFAULT_PROFILE` in the project root readme).